xml/art_sle_ha_install_quick.xml

```diff
@@ -93,6 +90,86 @@
     works, we will put one of the nodes into standby mode and check if the
     virtual IP address is migrated to the second node.
    </para>
+   <para>
+    You can use the two-node cluster for testing purposes or as a minimal
+    cluster configuration that you can extended later on. Before using the
+    cluster in a production environment, modify it according to your
+    requirements.
+   </para>
+  </sect1>
+
+  <sect1 xml:id="sec.ha.inst.quick.workflow">
+   <title>Workflow for Setting Up the One-Node Cluster</title>
+   <remark>toms 2016-08-01: Not sure about the introducery para.</remark>
+   <para>
+    All commands from the <package>ha-cluster-bootstrap</package> package
+    execute bootstrap scripts that require only a minimum of time and manual
+    intervention.
+   </para>
+   <para>
+    The important bootstrap script <command>ha-cluster-init</command> use
+    the following components during setup of a one-node cluster:
+   </para>
+
+   <variablelist>
+    <varlistentry>
+     <term>NTP</term>
+     <listitem>
+      <para>
+       Checking if NTP has not been configured to start at boot time, a
+       message appears.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>SSH</term>
+     <listitem>
+      <para>Creation of SSH keys for passwordless login between cluster nodes.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>&csync;</term>
+     <listitem>
+      <para>
+       Configuration of &csync; to replicate configuration files accross all nodes
+       in a cluster.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>&corosync;</term>
+     <listitem>
+      <para>Configuration of the cluster communication system.</para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>SBD/Watchdog</term>
+     <listitem>
+      <para>Configuration of the node fencing mechanism (SBD) and hardware
```

Comment:
It checks if a watchdog exists and asks you if to configure SBD as node fencing mechanism.